assets/openstack/credentials_request.yaml

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
I think this should belong to a regular `manifests` folder. cloud-credentials-operator will filter which credentials to pass according to current cloud provider.